ipld/resolver/src/service.rs

```diff
@@ -160,35 +161,29 @@ where
     {
         let peer_id = config.network.local_peer_id();
         let transport = transport(config.network.local_key.clone());
+
+        // NOTE: Hardcoded values from Forest. Will leave them as is until we know we need to change.
+        let limits = ConnectionLimits::default()
+            .with_max_pending_incoming(Some(10))
+            .with_max_pending_outgoing(Some(30))
+            .with_max_established_incoming(Some(config.connection.max_incoming))
+            .with_max_established_outgoing(None) // Allow bitswap to connect to subnets we did not anticipate when we started.
+            .with_max_established_per_peer(Some(5));
+
         let behaviour = Behaviour::new(
             config.network,
             config.discovery,
             config.membership,
             config.content,
+            limits,
             store,
         )?;
 
-        // NOTE: Hardcoded values from Forest. Will leave them as is until we know we need to change.
+        let swarm_config = libp2p::swarm::Config::with_tokio_executor()
+            .with_notify_handler_buffer_size(std::num::NonZeroUsize::new(20).expect("Not zero"))
+            .with_per_connection_event_buffer_size(64);
 
-        // TODO: Where this these go? Used to be `SwarmBuilder::connection_limits`
```

Comment:
I think it looks good, but out of curiosity, how do you interpret this comment? I’m wondering why it was commented out.

Reply:
The person migration did not know where this functionality migrated to.